archival-pipeline/extract.py

```diff
@@ -0,0 +1,67 @@
+'''This is an extract script for the archive pipeline. it connects the RDS database 
+and converts the data into a pandas dataframe'''
+import os
+from datetime import datetime, timedelta
+import psycopg2
+from psycopg2.extensions import connection
+import pandas as pd
+from dotenv import load_dotenv
+
+
+class DatabaseManager:
+    '''Handles database connection and queries to the RDS.'''
+
+    FETCH_ARTICLE_DATA_QUERY = """
+            SELECT
+                a.article_id, at.article_topic_id, a.article_headline, a.article_url,
+                a.article_published_date, a.article_subjectivity, a.article_polarity,
+                no.news_outlet_name, t.topic_name, at.article_topic_positive_sentiment, 
+                at.article_topic_negative_sentiment,at.article_topic_neutral_sentiment, 
+                at.article_topic_compound_sentiment
+            FROM article_topic at
+            JOIN article a ON a.article_id = at.article_id
+            JOIN topic t ON t.topic_id = at.topic_id
+            JOIN news_outlet no ON no.news_outlet_id = a.news_outlet_id
+            WHERE a.article_published_date < %s
+        """
+
+    def __init__(self) -> None:
+        '''Initializes the DatabaseManager by connecting to the RDS database.'''
+        load_dotenv()
+        self.db_connection = self._create_connection()
```

Comment:
Super nit-picky, but I would make this attribute private.
```suggestion
        self.__db_connection = self._create_connection()
```